$OpenBSD: configure_in,v 1.1.1.1 2001/07/13 15:28:19 avsm Exp $
Do not apply this patch, since it triggers off an automake run.
Useful to keep here to regenerate the configure patch in the future
--- configure.in.orig	Thu Jun 14 20:29:13 2001
+++ configure.in	Thu Jun 14 20:31:49 2001
@@ -302,8 +302,8 @@ AC_CHECK_HEADER(freetype/freetype.h,[
 	AC_EGREP_HEADER(FT_Library,freetype/freetype.h,[
 		AC_MSG_RESULT(yes)
 		have_ft_header=yes
-		WMF_CFLAGS="$WMF_CFLAGS $freetype_cflags"
-		WMF_CONFIG_CFLAGS="$WMF_CONFIG_CFLAGS $freetype_cflags"
+		WMF_CFLAGS="$freetype_cflags $WMF_CFLAGS"
+		WMF_CONFIG_CFLAGS="$freetype_cflags $WMF_CONFIG_CFLAGS"
 	],[	AC_MSG_RESULT(no)
 		have_ft_header=no
 	])
